newIDE/app/src/ObjectsList/index.js

```diff
@@ -503,6 +503,7 @@ export default class ObjectsListContainer extends React.Component<
             }
             onCancel={() => this._editVariables(null)}
             onApply={() => this._editVariables(null)}
+            titleMessage="Object Variables:"
```

Comment:
"Object Variables"

Reply:
@blurymind You've not actually removed the colon ;)

Reply:
oops,sorry I missed it :)